app/taskroutes.py

```diff
@@ -0,0 +1,148 @@
+from flask import Blueprint, jsonify, abort, make_response, request
+from app.models.task import Task
+from app.models.goal import Goal
+from app import db
+from sqlalchemy import asc, desc
+import datetime, requests, os
+from dotenv import load_dotenv
+
+load_dotenv()
```

Comment:
This is usually something that goes in the `__init__` file and, in fact code in there already calls `load_dotenv`. I cloned your repository just to see if there was something I was missing, but for sure, this works fine with this line commented out.